BenadrylAutomation/src/test/java/com/jnj/testcases/BenadrylUsesTest.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * @author Goutam Naik
+ * @date 12-Mar-24
+ */
+
+package com.jnj.testcases;
+
+import org.testng.SkipException;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import com.jnj.base.BaseClass;
+import com.jnj.dataproviders.BenadrylUsesProvider;
+
+public class BenadrylUsesTest extends BaseClass {
+	@BeforeMethod
+	public void setup() throws InterruptedException {
+		launchApplication();
+	}
+
+	@Test(priority = 1, dataProvider = "banner", dataProviderClass = BenadrylUsesProvider.class)
+	public void USES_verifyBannerContent(String testcase, String execution, String menu, String subMenu, String heading, String description) throws InterruptedException {
+		test = test.createNode(testcase);
+		if (execution.equalsIgnoreCase(defaultFlag)) {
+			selectEnv(runOn);
+			home.closePrivacyPopup();
+			header.navigateSubMenuLink(menu, subMenu);
+			signup.closeSignUpPopup();
+			uses.verifyMainHeaderInfo(heading, description);
+		} else {
+			throw new SkipException("Test skipped : " + testcase);
+		}
+	}
+	
+	@Test(priority = 2, dataProvider = "seasonalAllergies", dataProviderClass = BenadrylUsesProvider.class)
+	public void USES_verifySeasonalAllergies(String testcase, String execution, String menu, String subMenu, String expectedHeading, String expectedDescription, String expectedUrl) throws InterruptedException {
+		test = test.createNode(testcase);
+		if (execution.equalsIgnoreCase(defaultFlag)) {
+			selectEnv(runOn);
+			home.closePrivacyPopup();
+			header.navigateSubMenuLink(menu, subMenu);
+			signup.closeSignUpPopup();
+			uses.verifySeasonalAllergies(expectedHeading, expectedDescription, expectedUrl);
+		} else {
+			throw new SkipException("Test skipped : " + testcase);
+		}
+	}
+	
+	@Test(priority = 3, dataProvider = "allergiesDesc", dataProviderClass = BenadrylUsesProvider.class)
+	public void USES_verifyAllergiesDesc(String testcase, String execution, String menu, String subMenu, String section, String heading, String description) throws InterruptedException {
+		test = test.createNode(testcase);
+		if (execution.equalsIgnoreCase(defaultFlag)) {
+			selectEnv(runOn);
+			home.closePrivacyPopup();
+			header.navigateSubMenuLink(menu, subMenu);
+			signup.closeSignUpPopup();
+			uses.verifyAllergiesDesc(section, heading, description);
+		} else {
+			throw new SkipException("Test skipped : " + testcase);
+		}
+	}
+	
+	@Test(priority = 3, dataProvider = "symptoms", dataProviderClass = BenadrylUsesProvider.class)
```

Comment:
@goutamnaik007 priority value repeated.

Reply:
Priorities of the test and the tests following it are updated now.
@vaibhavNagvekarSJI